.github/workflows/prerelease.yml

```diff
@@ -15,7 +15,14 @@ jobs:
       - uses: actions/setup-node@v1
       - name: Install dependencies
         run: npm install
+      - name: Build
+        run: npm run build
       - name: Set registry
         run: 'echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc'
       - name: Publish pre-release of @manifoldco/mui-core 🤫
         run: 'npm publish --tag next'
+      - name: Publish pre-release of @manifoldco/mui-core-types 🤫
+        run: |
+          cp -r ./dist/types/v* ./mui-core-types/
+          ls -l ./mui-core-types 
+          npm publish mui-core-types/ --tag next
```

Comment:
Trying to publish types here: https://github.com/manifoldco/mui-core/pull/16/checks?check_run_id=464833003

As you can see the copy command is running but when I list the files in `mui-core-types` it only shows me the `package.json` that's alrready in there. As a result, this is the only file that gets published :/

Reply:
Does `ls -l ./dist/types/v*` show you the files that you expect to see?

Reply:
I'll try that

Reply:
ok wut...
```
total 12
-rw-r--r-- 1 runner docker 591 Feb 24 14:58 ManifoldError.d.ts
-rw-r--r-- 1 runner docker 864 Feb 24 14:58 graphqlFetch.d.ts
-rw-r--r-- 1 runner docker 462 Feb 24 14:58 index.d.ts
total 8
-rw-r--r-- 1 runner docker  269 Feb 24 14:58 package.json
drwxr-xr-x 2 runner docker 4096 Feb 24 14:58 v0
```
the files are there, and they're being copied now lol 😅

Reply:
oh, never mind. the types were being copied fine. there was some garbage in between the 2 list items before for some reason. they're being published :)